src/app/handlers/membercount.handler.ts

```diff
@@ -0,0 +1,62 @@
+import { GuildMember, MessageEmbed, TextChannel } from 'discord.js';
+import Constants from '../../common/constants';
+import { IContainer, IHandler, IServerInfo } from '../../common/types';
+
+export class MemberCountHandler implements IHandler {
+  private _memberMilestoneInterval = 100;
+
+  constructor(public container: IContainer) {}
+  public async execute(member: GuildMember) {
+    const collections = await this.container.storageService.getCollections();
+    const { serverInfo: serverInfoCollection } = collections;
+    const knightEmoji = member.guild.emojis.cache.find((e) => e.name === 'knight');
+
+    if (!serverInfoCollection) {
+      return;
+    }
+
+    const res = await serverInfoCollection
+      .find<IServerCount>({ name: 'MemberCount' })
+      .toArray();
```

Comment:
why are you keeping an array?

Reply:
Why have more than one entry of the record?
I was planning on storing it, so later on we can see the growth overtime, maybe eventually make a graph or something